packages/evo-sdk-common/src/evo/common/connector.py

```diff
@@ -235,6 +236,8 @@ async def call_api(
             response_type = default_response_type
 
         # Decode response.
+        logger.debug("RAW RESPONSE")
+        logger.debug(str(response.data))
```

Comment:
I also prefer not to split log messages that should be formatted together, because it is easy for them to get separated in the log output stream.

```suggestion
        logger.debug(f"HTTP response {response.status} ({response.reason}):\n{response.data}")
```

Food for thought:
* in the specific case of request + response, do we want to generate a correlation ID that we can include in debug messages?
* is it too much to include the raw response data in debug messages? Would it be sufficient to log the status code (and reason)?
* * `ApiConnector` is also used to transfer binary data, which can potentially be GB worth of bytes in the log output in this case